java/core/src/test/java/co/worklytics/psoxy/rules/msft/TeamsTests.java

```diff
@@ -0,0 +1,473 @@
+    public void users(String apiVersion) {
+        String endpoint = "https://graph.microsoft.com/" + apiVersion + "/users";
+        String jsonResponse = asJson("Users_"+ apiVersion + ".json");
+        assertNotSanitized(jsonResponse,
+            "jobTitle",
+            "officeLocation",
+            "preferredLanguage",
+            "id","6ea91a8d-e32e-41a1-b7bd-d2d185eed0e0",
+            "id","4562bcc8-c436-4f95-b7c0-4f8ce89dca5e"
+        );
+
+        String sanitized = sanitize(endpoint, jsonResponse);
+        assertRedacted(sanitized,
+            "businessPhones", "425-555-0100",
+            "displayName", "Conf Room Adams",
+            "givenName",
+            "mail", "Adams@contoso.com",
+            "mobilePhone", "234-567-0989",
+            "userPrincipalName", "Adams@contoso.com"
+        );
+        assertUrlWithSubResourcesBlocked(endpoint);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"v1.0", "beta"})
+    @Description("Test endpoint: " + PrebuiltSanitizerRules.MS_TEAMS_PATH_TEMPLATES_USERS_ONLINE_MEETINGS)
+    public void users_onlineMeetings(String apiVersion) {
+        String userId = "dc17674c-81d9-4adb-bfb2-8f6a442e4622";
+        String endpoint = "https://graph.microsoft.com/" + apiVersion + "/users/" + userId + "/onlineMeetings";
+        String jsonResponse = asJson("Users_onlineMeetings_"+ apiVersion + ".json");
+        assertNotSanitized(jsonResponse,
+            "everyone",
+            "5552478",
+            "5550588",
+            "9999999",
+            "https://dialin.teams.microsoft.com/6787A136-B9B8-4D39-846C-C0F1FF937F10?id=xxxxxxx",
+            "153367081",
+            "2018-05-30T00:12:19.0726086Z",
+            "2018-05-30T01:00:00Z",
+            "112f7296-5fa4-42ca-bae8-6a692b15d4b8_19:cbee7c1c860e465f8258e3cebf7bee0d@thread.skype",
+            "https://teams.microsoft.com/l/meetup-join/19%3a:meeting_NTg0NmQ3NTctZDVkZC00YzRhLThmNmEtOGQDdmZDZk@thread.v2/0?context=%7b%22Tid%22%3a%aa67bd4c-8475-432d-bd41-39f255720e0a%22%2c%22Oid%22%3a%22112f7296-5fa4-42ca-bb15d4b8%22%7d",
+            "112f7296-5ca-bae8-6a692b15d4b8",
+            "5810cedeb-b2c1-e9bd5d53ec96",
+            "joinMeetingId", "1234567890"
+        );
+
+        String sanitized = sanitize(endpoint, jsonResponse);
+        assertRedacted(sanitized,
+            "Tyler Stein",
+            "Jasmine Miller",
+            "Test Meeting.",
+            "passcode",
+            "isPasscodeRequired",
+            "127.0.0.1",
+            "macAddress",
+            "reflexiveIPAddress",
+            "relayIPAddress",
+            "subnet"
+        );
+        assertUrlWithSubResourcesBlocked(endpoint);
+    }
+
+    @Override
+    public Stream<InvocationExample> getExamples() {
+        String apiVersion = "v1.0";
+
+        return Stream.of(
+            InvocationExample.of("https://graph.microsoft.com/" + apiVersion + "/teams", "Teams_" + apiVersion + ".json"),
```

Comment:
As *apiVersion* is the same, you could extract it and use it as variable to save some text in the list: 

```
String baseEndpoint = "https://graph.microsoft.com/" + apiVersion
```